lib/src/container.dart

```diff
@@ -19,23 +19,26 @@ class PageIndicatorContainer extends StatefulWidget {
 
   final IndicatorAlign align;
 
+  final IndicatorShape shape;
+
   final Color indicatorColor;
 
   final Color indicatorSelectorColor;
 
   final double indicatorSpace;
 
-  const PageIndicatorContainer({
-    Key key,
-    @required this.pageView,
-    @required this.length,
-    this.padding = const EdgeInsets.only(bottom: 15.0),
-    this.size = 12.0,
-    this.align = IndicatorAlign.bottom,
-    this.indicatorColor = Colors.white,
-    this.indicatorSelectorColor = Colors.grey,
-    this.indicatorSpace = 8.0,
-  }) : super(key: key);
+  const PageIndicatorContainer(
+      {Key key,
+      @required this.pageView,
+      @required this.length,
+      this.padding = const EdgeInsets.only(bottom: 15.0,top: 8.0),
```

Comment:
Added Top padding here

Reply:
Maybe I didn't make it clear. I mean:
If the bottom of padding is 0 and align is bottom, there should be no blank at the bottom of Indicator.

like this:

![image](https://user-images.githubusercontent.com/14145407/56270548-fc3db780-6128-11e9-907d-bf51ce2c0558.png)

Reply:
I have modified this part of the code. If you agree, I will upload this part of the code to your pr.

Reply:
Please do... :)